custom_components/places/attributes.py

```
@@ -0,0 +1,178 @@
+        """Return whether a value is considered blank.
+
+        Args:
+            attr: Attribute key to evaluate.
+
+        Returns:
+            ``True`` for missing values, ``None`` and empty string values. Numeric
+            zero is treated as non-blank for compatibility with prior behavior.
+        """
+        if self._internal_attr.get(attr) or self._internal_attr.get(attr) == 0:
+            return False
+        return True
```

Ruff (0.15.12)

[warning] Return the negated condition directly

Inline condition

(SIM103)